bootstrap/sql/migrations/native/1.5.0/mysql/schemaChanges.sql

```diff
@@ -232,3 +232,55 @@ UPDATE ingestion_pipeline_entity
+update glossary_term_entity set json = JSON_REMOVE(json, '$.owner') where json -> '$.owner' is not null;
+update ingestion_pipeline_entity set json = JSON_REMOVE(json, '$.owner') where json -> '$.owner' is not null;
+update kpi_entity set json = JSON_REMOVE(json, '$.owner') where json -> '$.owner' is not null;
+update messaging_service_entity set json = JSON_REMOVE(json, '$.owner') where json -> '$.owner' is not null;
+update metadata_service_entity set json = JSON_REMOVE(json, '$.owner') where json -> '$.owner' is not null;
+update metric_entity set json = JSON_REMOVE(json, '$.owner') where json -> '$.owner' is not null;
+update ml_model_entity set json = JSON_REMOVE(json, '$.owner') where json -> '$.owner' is not null;
+update mlmodel_service_entity set json = JSON_REMOVE(json, '$.owner') where json -> '$.owner' is not null;
+update persona_entity set json = JSON_REMOVE(json, '$.owner') where json -> '$.owner' is not null;
+update pipeline_entity set json = JSON_REMOVE(json, '$.owner') where json -> '$.owner' is not null;
+update pipeline_service_entity set json = JSON_REMOVE(json, '$.owner') where json -> '$.owner' is not null;
+update policy_entity set json = JSON_REMOVE(json, '$.owner') where json -> '$.owner' is not null;
+update query_entity set json = JSON_REMOVE(json, '$.owner') where json -> '$.owner' is not null;
+update report_entity set json = JSON_REMOVE(json, '$.owner') where json -> '$.owner' is not null;
+update role_entity set json = JSON_REMOVE(json, '$.owner') where json -> '$.owner' is not null;
+update search_index_entity set json = JSON_REMOVE(json, '$.owner') where json -> '$.owner' is not null;
+update search_service_entity set json = JSON_REMOVE(json, '$.owner') where json -> '$.owner' is not null;
+update storage_container_entity set json = JSON_REMOVE(json, '$.owner') where json -> '$.owner' is not null;
+update storage_service_entity set json = JSON_REMOVE(json, '$.owner') where json -> '$.owner' is not null;
+update stored_procedure_entity set json = JSON_REMOVE(json, '$.owner') where json -> '$.owner' is not null;
+update table_entity set json = JSON_REMOVE(json, '$.owner') where json -> '$.owner' is not null;
+update team_entity set json = JSON_REMOVE(json, '$.owner') where json -> '$.owner' is not null;
+update thread_entity set json = JSON_REMOVE(json, '$.owner') where json -> '$.owner' is not null;
+update topic_entity set json = JSON_REMOVE(json, '$.owner') where json -> '$.owner' is not null;
+update type_entity set json = JSON_REMOVE(json, '$.owner') where json -> '$.owner' is not null;
+update user_entity set json = JSON_REMOVE(json, '$.owner') where json -> '$.owner' is not null;
+
+update table_entity set json = JSON_SET(
+    JSON_REMOVE(json, '$.dataModel.owner'),
+    '$.dataModel.owners',
+    JSON_ARRAY(
+        JSON_EXTRACT(json, '$.dataModel.owner')
+    )
+) where json -> '$.dataModel.owner' is not null;
```

Comment:
Does the order here of Removing and creating the array later work?

(I'm not 100% familiar with JSON Operations on MySQL and I'm asking just in case)

Reply:
Yes, as the rename doesn't work as expected for json, we need to remove and add

so the process here is:
-     JSON_REMOVE(json, '$.dataModel.owner'),
to return json without owner
-     '$.dataModel.owners',
the name of the key we want to add it in
- JSON_ARRAY(
        JSON_EXTRACT(json, '$.dataModel.owner')
    )
    
    what to add it in


it won't remove until it gets the whole json and then sets it.